### DISCUSSION TITLE
ERROR in https://github.com/coreybutler/nvm-windows/wiki

### DISCUSSION THREAD

**Discussion by DingBEN** [coreybutler/nvm-windows#1038] [General]
Leaving a simple message is HELL... wtf!
3rd time typing the same info - Please pass this message to Corey Butler

https://github.com/coreybutler/nvm-windows/wiki page states the nvm.setup.zip contains the nvm.setup.exe but instead it contains nvm.UPDATAE.exe (did nothing, no error, no msg, runs for a millisecond and gagaland!)

I had to download the separate nvm.setup.exe to setup on my WIN7.
I verified that ALL the parameters listed in the manual installation instruction on same page... all was flawless.
Have not tested nvm yet...

**Comment by coreybutler** [comment]
nvm-setup.zip and nvm-update.zip both contain the appropriate files, as shown below:

<img width="926" alt="image" src="https://github.com/coreybutler/nvm-windows/assets/770982/0382cdf0-367c-47f0-944e-fe2f08223aba">

Microsoft ceased support for Windows 7 in Jan 2020. While NVM4W _may_ work, it is not a tested/supported OS. That operating system is drastically different from Windows 8+ in how it handles symlinks and elevated permissions.. enough so that I cannot and will not support it (it's basically writing/maintaining a separate app). That said, I do hope it works for you. I'm 99.999% certain you _must_ run NVM4W as an admin, because no other users have symlink permissions in Windows 7 (unless granted through an AD GPO).

**Comment by DingBEN** [comment]
thank you very much for this thorough answer
I was just a my *WIN7* to do a quick test,
I plan to run nvm on a *WIN10* unit!
